container/container.go

```diff
@@ -105,26 +106,65 @@ func (c *Container) FindContainer(namespace []string) (types.ContainerJSON, erro
 	return c.client.ContainerInspect(ctx, containerID)
 }
 
-// Status returns the status of a docker container and servcie.
+// Status returns the status of the container based on the docker container and docker service
+// If any error happen the status will be `UNKNOWN`
+// Otherwise the following status will be applied:
+//   - RUNNING: When the container is running in docker (whatever the status of the service)
+//   - STARTING: When the service is running but the container is not yet started
+//   - STOPPED: When neither the container nor the service are running in docker
 func (c *Container) Status(namespace []string) (StatusType, error) {
```

Comment:
```
// status returns the status of the container based on the docker container and docker service.
// if any error occurs during the status check, status will be shown as UNKNOWN.
// otherwise the following rules will be applied to determine status:
//  - RUNNING: when the container is running in docker regardless of the status of the service.
//  - STARTING: when the service is running but the container is not yet started.
//  - STOPPED: when the container and the service is not running in docker.
```